package.json

```diff
@@ -52,17 +52,17 @@
   },
   "dependencies": {
     "@ledgerhq/compressjs": "^1.3.2",
-    "@ledgerhq/devices": "5.29.0",
-    "@ledgerhq/errors": "5.29.0",
-    "@ledgerhq/hw-app-btc": "5.29.0",
-    "@ledgerhq/hw-app-eth": "5.29.0",
-    "@ledgerhq/hw-app-xrp": "5.29.0",
-    "@ledgerhq/hw-transport": "5.29.0",
+    "@ledgerhq/devices": "5.30.0",
+    "@ledgerhq/errors": "5.30.0",
+    "@ledgerhq/hw-app-btc": "5.30.0",
+    "@ledgerhq/hw-app-eth": "5.30.0",
+    "@ledgerhq/hw-app-xrp": "5.30.0",
+    "@ledgerhq/hw-transport": "5.30.0",
     "@ledgerhq/hw-transport-http": "5.29.0",
-    "@ledgerhq/live-common": "16.0.0-countervalues.1",
-    "@ledgerhq/logs": "5.29.0",
-    "@ledgerhq/react-native-hid": "5.29.0",
-    "@ledgerhq/react-native-hw-transport-ble": "5.29.0",
+    "@ledgerhq/live-common": "17.0.0",
+    "@ledgerhq/logs": "5.30.0",
+    "@ledgerhq/react-native-hid": "5.30.0",
+    "@ledgerhq/react-native-hw-transport-ble": "5.30.0",
```

Comment:
(HODL this PR and wait for live-common to get merged because live-common 17.0.0 have bug on xrp)